__tests__/app.test.js

```diff
@@ -196,7 +196,24 @@ describe('POST /api/articles/:article_id/comments', () => {
               comment_id: 19}))
       })
     })
-    test('should respond with 400 error if body is not in correct format', () => {
+    test('should respond with the posted comment, ignoring additional properties on the request body', () => {
+      const newComment = {username: "butter_bridge", body: "test comment", extra: "ignore this"}
+      return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toEqual(
```

Comment:
I think to object containing will not error if there are additional properties on the tested object? Think it only errors when any of the listed ones are missing from the tested object. So here we want to explicitly check that the extra' property _doesnt_ exist on the object in addition to the fact that the other ones do